tools/db_bench_tool.cc

```diff
@@ -4592,6 +4593,12 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         abort();
       }
 
+      if (thread->shared->write_rate_limiter) {
+        thread->shared->write_rate_limiter->Request(
+            key.size() + value_size_, Env::IO_HIGH, nullptr /*stats*/,
+            RateLimiter::OpType::kWrite);
+      }
```

Comment:
May be I am wrong, but don't you need to check `FLAGS_benchmark_write_rate_limit` also and handle it, and not just if `thread->shared->write_rate_limiter` is not null?